src/NuGet.Core/NuGet.Packaging/PackageExtractor.cs

```diff
@@ -549,7 +549,7 @@ public static class PackageExtractor
 
                             File.Move(tempNupkgMetadataPath, nupkgMetadataFilePath);
 
-                            logger.LogInformation(StringFormatter.Log_InstalledPackage(packageIdentity.Id, packageIdentity.Version.OriginalVersion, source, contentHash));
+                            logger.LogInformation(StringFormatter.Log_InstalledPackage(packageIdentity.Id, packageIdentity.Version.OriginalVersion, source, contentHash, nupkgMetadataFilePath));
```

Comment:
I think we want the package path folder. People shouldn't care about the nupkg metadata.

Reply:
Fixed